jupyterhub/utils.py

```diff
@@ -115,24 +118,31 @@ def decorated(self, *args, **kwargs):
     decorator.__doc__ = check_auth.__doc__
     return decorator
 
+
 @auth_decorator
 def token_authenticated(self):
-    """decorator for a method authenticated only by the Authorization token header
+    """Decorator for method authenticated only by Authorization token header.
 
     (no cookies)
     """
     if self.get_current_user_token() is None:
         raise web.HTTPError(403)
 
+
 @auth_decorator
 def authenticated_403(self):
-    """like web.authenticated, but raise 403 instead of redirect to login"""
+    """Decorator for method to raise 403 error instead of redirect to login.
```

Comment:
I could go either way on this. The periods are a convention in pep8 and numpy docstrings. I think it's fine without the period too. In general, I'm less concerned about the punctuation and more focused on having at short, descriptive docstrings :-)